examples/unstructured_2d_dgsem/elixir_euler_sedov.jl

```diff
@@ -1,5 +1,4 @@
 
-using Downloads: download
 using OrdinaryDiffEq
```

Comment:
I can't comment on the relevant line below: Where do we get the 
```
default_mesh_file = joinpath(@__DIR__, "mesh_periodic_square_with_twist.mesh")
```
from?

Reply:
Apparently, this is downloaded in three other elixirs, and the download line is just missing here. I guess no one noticed it because it never got executed before the other elixirs.

Reply:
Could you also fix it in this PR, please?